docs/howtoguides/enable_cc.md

```diff
@@ -0,0 +1,20 @@
+# How to enable CC EAL
+
+Common Criteria is supported only on 16.04 and 18.04. For more information on it,
```

Comment:
Maybe we can insert a comment saying that the certified versions are `16.04.4` and `18.04.4`, stating that the scripts will be there anyway but will only run on those point releases